libraries/Microsoft.Bot.Builder.Runtime/Microsoft.Bot.Builder.Runtime.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version Condition=" '$(ReleasePackageVersion)' == '' ">$(LocalPackageVersion)</Version>
```

Comment:
You may want to ensure we use PreviewPackageVersion until this is ready for prime time. 

![image](https://user-images.githubusercontent.com/12264946/99291769-4f4c8480-280e-11eb-94cf-7886517c14aa.png)